backend/s3/s3.go

```diff
@@ -1381,6 +1393,19 @@ func (o *Object) split() (bucket, bucketPath string) {
 	return o.fs.split(o.remote)
 }
 
+// getClient makes an http client according to the options
+func getClient(opt *Options) *http.Client {
+	// TODO: Do we need cookies too?
```

Comment:
Cookies?

Reply:
@ncw Do we need the cookies too btw? I noticed that even for Google drive the cookie jar is not set so am not sure whether we need that or not?

Reply:
I don't think we need cookies.